src/sqleet-v0.31.1/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.2)
```

Comment:
Probably not for this PR, but it would be great to pull sqleet from a separated conan repository. This way, source code for this library wouldn't be duplicated in this repo and it would be better for future upgrades of sqleet. In order to do so, I would recommend forking the sqleet repo and adding there the conan receipe needed to pack the library.

Reply:
Yes, I totally agree with that, but as you comment it required an additional effort now. 
I will add an issue to the repository for this.